src/proto.c

```diff
@@ -287,53 +287,6 @@ void proto_close(int prot_sock)
 	proto->close(prot_sock);
 }
 
-int proto_mknode(int proto_socket, const char *owner_uuid,
-		 const char *device_name, const char *device_id,
-		 char *uuid, char *token)
-{
-	json_object *device;
-	const char *device_as_string;
-	json_raw_t response;
-	int err, result;
-
-	memset(&response, 0, sizeof(response));
-	device = parser_device_json_create(owner_uuid, device_name, device_id);
-	if (!device) {
-		hal_log_error("JSON: no memory");
-		result = KNOT_ERR_INVALID;
-		goto fail;
-	}
-
-	device_as_string = json_object_to_json_string(device);
-	err = proto->mknode(proto_socket, device_as_string, &response);
-	json_object_put(device);
-
-	if (err < 0) {
-		hal_log_error("manager mknode: %s(%d)", strerror(-err), -err);
-		result = KNOT_ERR_CLOUD_FAILURE;
-		goto fail;
-	}
-
-	if (response.size == 0 ||
-	    (parser_device(response.data, uuid, token) < 0)) {
```

Comment:
This function is not been used anymore. Maybe you should call it in another place or remove it.

Reply:
Done